src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,97 @@
+package duke;
```

Comment:
Your variable names are intuitive and are following the coding standard. Nicely done!